app/angular/jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   preset: 'jest-preset-angular',
   setupFilesAfterEnv: ['<rootDir>/setup-jest.ts'],
+  moduleNameMapper: {
+    '@storybook/node-logger': '<rootDir>/../../lib/node-logger/dist/cjs/index.js',
```

Comment:
I honestly don't know how jest managed to successfully run tests with external packages without `moduleNameMapper` 🤔

Reply:
Now `build-storybook` fails with 

```
ERR! Module parse failed: Unexpected character '@' (11:61)
```

I guess that's not the solution we were looking for